### PR TITLE
Happy Thoughts API

### PR DESCRIPTION
**Pull request by Jenfi**
Heroku-link: https://jennifershappythoughts.herokuapp.com/
Deployed front-end: https://jennifershappythoughts.netlify.com/

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -7,23 +7,66 @@ const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/happyThoughts"
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
 mongoose.Promise = Promise
 
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
+const Thought = mongoose.model('Thought', {
```
Comment (JennieDalgren) [inline server.js]: ⭐️

File: server.js [hunk 2]
```diff
@@ -7,23 +7,66 @@ const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/happyThoughts"
...
+  try {
+    const { message } = req.body
+    const thought = await new Thought({ message }).save();
+    res.status(200).json(thought);
+  } catch (err) {
+    res.status(400).json({ message: 'Could not post thought', errors: err.errors })
+  }
+});
+
+app.post('/:thoughtId/like', async (req, res) => {
```
Comment (JennieDalgren) [inline server.js]: ⭐️

File: server.js [hunk 3]
```diff
@@ -7,23 +7,66 @@ const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/happyThoughts"
...
+  res.json(thoughts);
+})
+
+app.get('/:thoughtId', async (req, res) => {
+  const thoughtId = req.params.thoughtId
+  Thought.findOne({ '_id': thoughtId })
+    .then((results) => {
+      res.json(results)
+    })
+})
```
Comment (JennieDalgren) [inline server.js]: what is this endpoint used for?

Comment (Jenfi) [reply server.js]: Hi Jennie, this endpoint was for getting a specific thought, but it is not very useful, so I guess I could remove it :) 

I can see that you have managed to post comments to the API, which I also did last Friday, but now for some reason I cannot post it... Is there any chance you can see why?

**Pull request closed without merging**